components/contact/ContactForm.jsx

```diff
@@ -1,14 +1,36 @@
 import React, { useState, useEffect, useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { toast } from 'react-toastify';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
`import makeStyles from '@mui/styles/makeStyles';` and remove the material-ui dependency

Reply:
thanks I went ahead and changed this to your suggestion